src/main/java/de/rabea/controller/NotFoundController.java

```diff
@@ -0,0 +1,13 @@
+package de.rabea.controller;
+
+import de.rabea.Controller;
+import de.rabea.request.HttpRequest;
+import de.rabea.response.HttpResponse;
+
+public class NotFoundController implements Controller {
+
+    @Override
+    public HttpResponse getResponse(HttpRequest request) {
+        return new HttpResponse("HTTP/1.1 404 Not Found");
```

Comment:
Yeah, I know the way the response is hard coded is not great currently. I want to extract that into a different class and/or use an Enum for the response header. I'll look at that today.